libmpv/opengl_cb.h

```diff
@@ -163,10 +179,22 @@ extern "C" {
  *
  * glMPGetNativeDisplay("wl") should return a Wayland "struct wl_display *".
  *
- * glMPGetNativeDisplay("drm") should return a DRM FD casted to intptr_t (note
- * that a 0 FD is not supported - if this can happen in your case, you must
- * dup2() it to a non-0 FD).
- *
+ * glMPGetNativeDisplay("drm-params") should return an mp_drm_params structure pointer:
+ */
+struct mp_drm_params {
```

Comment:
Rename to `mpv_opengl_cb_drm_params`. Identifiers starting with `mp_` are always private to mpv.